app/src/api/useShorteningApi.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import api from "./api";
+
+export const useShorteningApi = (url) => {
+  const [linksList, setLinksList] = useState([]);
+
+  const getApi = async () => {
+    try {
+      const response = await api.get(`/shorten?url=${url}`);
+      setLinksList([
+        ...linksList,
+        { longLink: url, shortLink: response.data.result.short_link },
+      ]);
+      localStorage.setItem(
+        "list",
+        JSON.stringify([
+          ...linksList,
+          { longLink: url, shortLink: response.data.result.short_link },
+        ])
+      );
+    } catch (err) {
+      if (err.response) {
```

Comment:
Is it possible for error not to have the response? Why is this check needed?

Reply:
fixed